### DISCUSSION TITLE
How to fully remove an Project

### DISCUSSION THREAD

**Discussion by xstable** [kimai/kimai#4158] [Questions & Answers]
For Testing purposes, I've created a Project `Test ABC`, and now I'd like to remove it. 
But I didn't found any way to do it. 

Any suggestion how to do this, or isn't there any way to remove a project yet?

**Comment by xstable** [comment]
Ah, I've found it, there is only one place in the Project listing to archive this. 

![image](https://github.com/kimai/kimai/assets/1778068/e1ba89fd-557f-4e59-b34a-15c63d765856)

I'd like to suggest to also place an RED-Button: `DELETE this Project` into the Project-Overview and/or EDIT-Section of the Project: 

![image](https://github.com/kimai/kimai/assets/1778068/9b631bc2-871b-4c02-adee-9d024e981b0c)


![image](https://github.com/kimai/kimai/assets/1778068/bfeaabdb-63e5-4c5a-9638-780cc868decc)

**Reply by kevinpapst** [reply]
Deleting is usually not a good idea. The location of the button is consistent, always in the listing page.

I added it to the documentation https://www.kimai.org/documentation/project.html#deleting-a-project
